if_show_working_info: true #那些运行中的info正常提示或者什么的,要不要显示

if_simulink: false         ##后面会自动根据是否处于仿真，分配话题
########input_topic_part########配置传感器话题输入部分
odom_end_send_tf: false    #参数默认false，tf统一仅由优化端广播，调试odom_end时设置为true，则只由odom_end广播tf
matching_end_send_tf: false #参数默认false，tf统一仅由优化端广播，调试matching_end时设置为true，则只由matching_end广播tf



imu_topic: /livox/imu
scan_pointcloud_topic: /livox/lidar  #未去畸变原始点云
imu_link: laser_link
lidar_link: laser_link
car_base_link: laser_link                     #仿真车体连结点
init_position: [0.00,0.00,0.00]

sim_imu_topic: imu
sim_scan_pointcloud_topic: /velodyne_points2  #未去畸变原始点云
sim_imu_link: imu
sim_lidar_link: velodyne2_base_link
sim_car_base_link: base_link                     #仿真车体连结点
#初始，相对于点云pcd或者是gazebo仿真世界的位置
sim_init_position: [5.00,2.00,0.00]


undistorted_pointcloud_topic: /undistorted_points #设定完成去畸变点云的发布话题


base_frame_id:  map                   ##不要有斜杠
########input_topic_part########



registration_method: ICP_SVD # 里程计选择点云匹配方法 目前支持 ICP_SVD , ICP , NDT , NDT_CPU , SICP